### ISSUE TITLE
Training issue with divide by zero

### ISSUE THREAD

**Issue by rachel-bousfield** [buriburisuri/speech-to-text-wavenet#75]
Running train.py gives me the following message. What is happening here?

> Training started from epoch[000]-step[0].
/usr/local/lib/python2.7/site-packages/sugartensor/sg_train.py:333: RuntimeWarning: divide by zero encountered in remainder
  start_step = sess.run(tf.sg_global_step()) % opt.ep_size
I 0523:12:40:16.487:sg_train.py:301] 	Epoch[000:gs=0] - loss = NA
I 0523:12:40:16.489:sg_train.py:301] 	Epoch[001:gs=0] - loss = NA
I 0523:12:40:16.491:sg_train.py:301] 	Epoch[002:gs=0] - loss = NA
I 0523:12:40:16.493:sg_train.py:301] 	Epoch[003:gs=0] - loss = NA
I 0523:12:40:16.494:sg_train.py:301] 	Epoch[004:gs=0] - loss = NA
I 0523:12:40:16.496:sg_train.py:301] 	Epoch[005:gs=0] - loss = NA
I 0523:12:40:16.498:sg_train.py:301] 	Epoch[006:gs=0] - loss = NA
I 0523:12:40:16.500:sg_train.py:301] 	Epoch[007:gs=0] - loss = NA
I 0523:12:40:16.502:sg_train.py:301] 	Epoch[008:gs=0] - loss = NA
I 0523:12:40:16.503:sg_train.py:301] 	Epoch[009:gs=0] - loss = NA
I 0523:12:40:16.505:sg_train.py:301] 	Epoch[010:gs=0] - loss = NA
I 0523:12:40:16.507:sg_train.py:301] 	Epoch[011:gs=0] - loss = NA
I 0523:12:40:16.509:sg_train.py:301] 	Epoch[012:gs=0] - loss = NA
I 0523:12:40:16.510:sg_train.py:301] 	Epoch[013:gs=0] - loss = NA
I 0523:12:40:16.514:sg_train.py:301] 	Epoch[014:gs=0] - loss = NA
I 0523:12:40:16.517:sg_train.py:301] 	Epoch[015:gs=0] - loss = NA
I 0523:12:40:16.519:sg_train.py:301] 	Epoch[016:gs=0] - loss = NA
I 0523:12:40:16.521:sg_train.py:301] 	Epoch[017:gs=0] - loss = NA
I 0523:12:40:16.523:sg_train.py:301] 	Epoch[018:gs=0] - loss = NA
I 0523:12:40:16.525:sg_train.py:301] 	Epoch[019:gs=0] - loss = NA
I 0523:12:40:16.527:sg_train.py:301] 	Epoch[020:gs=0] - loss = NA
I 0523:12:40:16.529:sg_train.py:301] 	Epoch[021:gs=0] - loss = NA
I 0523:12:40:16.531:sg_train.py:301] 	Epoch[022:gs=0] - loss = NA
I 0523:12:40:16.534:sg_train.py:301] 	Epoch[023:gs=0] - loss = NA
I 0523:12:40:16.536:sg_train.py:301] 	Epoch[024:gs=0] - loss = NA
I 0523:12:40:16.538:sg_train.py:301] 	Epoch[025:gs=0] - loss = NA
I 0523:12:40:16.541:sg_train.py:301] 	Epoch[026:gs=0] - loss = NA
I 0523:12:40:16.544:sg_train.py:301] 	Epoch[027:gs=0] - loss = NA
I 0523:12:40:16.546:sg_train.py:301] 	Epoch[028:gs=0] - loss = NA
I 0523:12:40:16.547:sg_train.py:301] 	Epoch[029:gs=0] - loss = NA
I 0523:12:40:16.549:sg_train.py:301] 	Epoch[030:gs=0] - loss = NA
I 0523:12:40:16.550:sg_train.py:301] 	Epoch[031:gs=0] - loss = NA
I 0523:12:40:16.552:sg_train.py:301] 	Epoch[032:gs=0] - loss = NA
I 0523:12:40:16.554:sg_train.py:301] 	Epoch[033:gs=0] - loss = NA
I 0523:12:40:16.556:sg_train.py:301] 	Epoch[034:gs=0] - loss = NA
I 0523:12:40:16.557:sg_train.py:301] 	Epoch[035:gs=0] - loss = NA
I 0523:12:40:16.559:sg_train.py:301] 	Epoch[036:gs=0] - loss = NA
I 0523:12:40:16.561:sg_train.py:301] 	Epoch[037:gs=0] - loss = NA
I 0523:12:40:16.562:sg_train.py:301] 	Epoch[038:gs=0] - loss = NA
I 0523:12:40:16.564:sg_train.py:301] 	Epoch[039:gs=0] - loss = NA
I 0523:12:40:16.566:sg_train.py:301] 	Epoch[040:gs=0] - loss = NA
I 0523:12:40:16.567:sg_train.py:301] 	Epoch[041:gs=0] - loss = NA
I 0523:12:40:16.569:sg_train.py:301] 	Epoch[042:gs=0] - loss = NA
I 0523:12:40:16.571:sg_train.py:301] 	Epoch[043:gs=0] - loss = NA
I 0523:12:40:16.572:sg_train.py:301] 	Epoch[044:gs=0] - loss = NA
I 0523:12:40:16.575:sg_train.py:301] 	Epoch[045:gs=0] - loss = NA
I 0523:12:40:16.577:sg_train.py:301] 	Epoch[046:gs=0] - loss = NA
I 0523:12:40:16.579:sg_train.py:301] 	Epoch[047:gs=0] - loss = NA
I 0523:12:40:16.581:sg_train.py:301] 	Epoch[048:gs=0] - loss = NA
I 0523:12:40:16.582:sg_train.py:301] 	Epoch[049:gs=0] - loss = NA
I 0523:12:40:16.584:sg_train.py:301] 	Epoch[050:gs=0] - loss = NA
I 0523:12:41:20.924:sg_train.py:368] Training finished at epoch[50]-step[0].

**Comment by bt9411** [comment]
mee too :P

**Comment by bt9411** [comment]
have u found a problem @Noah-Franks

**Comment by rachel-bousfield** [comment]
@batbaji I should have added that this message shows up at the end. I'm not sure why this isn't training.

> I 0525:15:19:15.186:sg_train.py:368] Training finished at epoch[50]-step[0].
*** Error in `python': free(): invalid pointer: 0x00000000014ee060 ***
Aborted

**Comment by bt9411** [comment]
did you trained with CPU ?

**Comment by rachel-bousfield** [comment]
@batbaji I did

**Comment by batbaji** [comment]
you should run first for preprocess.py

**Comment by rachel-bousfield** [comment]
I found the issue. This project does not support the GSM encoding for .wav files. Converting my corpora to pcm_s16le did the trick.
